frontend/app/models/paper-trail-version.js

```diff
@@ -0,0 +1,10 @@
+import Model, { attr, belongsTo } from "@ember-data/model";
+
+export default class PaperTrailVersion extends Model {
```

Comment:
maybe we should call this model Log in front- and backend.

Reply:
same for this one.